esp/scm/ws_logaccess.ecm

```diff
@@ -246,7 +246,7 @@ ESPenum LogAccessStatusCode : int
 ESPStruct LogAccessStatus
 {
     LogAccessStatusCode Code;
-    LogAccessStatusCode Messages;
+    ESParray<string> Messages;
```

Comment:
Strictly speaking this change should be versioned, but I don't know if it is worth the work. @jeclrsg would it cause any issues for your code if the interface changes without a version update? If the client code is released in sync with the back-end would there be any issue at all? Would the worst outcome be one point release with error messages not printing (if they even are now) and would that be a big deal?

I'm assuming there are no open-source users relying on this since it is such a recent change anyways, so ECL Watch is the only user we need to worry about.

Reply:
It wouldn't cause any problems. Using this health-check url functionality doesn't exist anywhere in ECL Watch yet. I only noticed the problem while attempting to use our tooling in https://github.com/hpcc-systems/visualization to bump the version of the JS library for the LogAccess service in order to potentially use it in ECL Watch.

Reply:
> Strictly speaking this change should be versioned, but I don't know if it is worth the work. @jeclrsg would it cause any issues for your code if the interface changes without a version update? If the client code is released in sync with the back-end would there be any issue at all? Would the worst outcome be one point release with error messages not printing (if they even are now) and would that be a big deal?
> 
> I'm assuming there are no open-source users relying on this since it is such a recent change anyways, so ECL Watch is the only user we need to worry about.

Agreed. I'll version the change if nothing else to document the change.